utils/avro-components-utils/src/main/scala/pl/touk/nussknacker/engine/avro/AvroUtils.scala

```diff
@@ -105,31 +110,81 @@ object AvroUtils extends LazyLogging {
     }
 
   /**
-    * It's a simply mapper scala Map[String, Any] to Avro GenericRecord
+    * It's a simply mapper scala Any to Avro Proper Data
     */
-  def createRecord(schema: Schema, data: Map[String, Any]): GenericRecord = {
-    def createValue(value: Any, schema: Schema): Any = (value, schema.getType) match {
-      case (map: Map[String@unchecked, _], Schema.Type.RECORD) =>
-          createRecord(schema, map)
-      case (l: List[_], Schema.Type.ARRAY) =>
-        l.map(createValue(_, schema)).asJava
-      case (map: Map[String@unchecked, _], Schema.Type.MAP) =>
-        map.mapValues(createValue(_, schema)).asJava
-      case (str: String, Schema.Type.ENUM) =>
-        new GenericData.EnumSymbol(schema, str)
+  def createAvroData(value: Any, schema: Schema): Any =
```

Comment:
It is this little, "no magic" helper that is substitution to well known `RecordBuilder`? How it is different from `BestEffortAvroEncoder`? When to use it and when is the limit that should be used `BestEffortAvroEncoder`?

Reply:
Hmm, maybe yu're right.. It doesn't have sens. I'll remove it.